iaso/api/org_unit_change_requests/filters.py

```diff
@@ -36,9 +37,11 @@ class OrgUnitChangeRequestListFilter(django_filters.rest_framework.FilterSet):
     potential_payment_ids = django_filters.CharFilter(
         method="filter_potential_payments", label=_("Potential Payment IDs (comma-separated)")
     )
+    source_version_id = django_filters.NumberFilter(method="filter_source_version_id", label=_("Source version ID"))
```

Comment:
You don't need an additional method here, you can just follow the relationships:

```suggestion
        source_version_id = django_filters.NumberFilter(field_name="org_unit__version", label=_("Source version ID"))
```

Also can you please add a test in https://github.com/BLSQ/iaso/blob/f528f4cfb71ca53b78c207737ccc9630f14eb26a/iaso/tests/api/org_unit_change_requests/test_filters.py#L11, it should be simple enough.